.env.test

```diff
@@ -0,0 +1,14 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Required to use the Trello REST API
+TRELLO_API_KEY=aG3nUin3LyR3alAp1K3y
+TRELLO_API_TOKEN=aG3nUin3LyR3alAp1t0k3n
+TRELLO_BOARD_ID=aG3nUin3LyR3alb0ArD1d
```

Comment:
Very very minor and I know this is obviously fake at a second glance but for half a second I still had to look - I'd go further and just write them as `fake-key` etc personally!